src/Lidar.cc

```diff
@@ -476,3 +499,15 @@ bool Lidar::HasConnections() const
 {
   return this->dataPtr->pub && this->dataPtr->pub.HasConnections();
 }
+
+//////////////////////////////////////////////////
+double Lidar::Clamp(double _range) const
+{
+  if (gz::math::isnan(_range))
+    return this->RangeMax();
+
+  if (std::isfinite(_range))
+    return gz::math::clamp(_range, this->RangeMin(), this->RangeMax());
```

Comment:
some here `std::clamp`?

Reply:
changed to `std::clamp`. [5c28cce](https://github.com/gazebosim/gz-sensors/pull/447/commits/5c28cce2df6f3a2a90ef857792acfe504eb47ec3)